### ISSUE TITLE
CORS error

### ISSUE THREAD

**Issue by Chabloz** [donmccurdy/aframe-inspector-plugin-recast#14]
Hi,

Thx for this very useful aframe component.  All was working fine until recently. Now, when a try to make an API call (to the public one), I've got the following  CORS error:

Access to fetch at 'https://recast-api.donmccurdy.com/v1/build/?cellSize=0.3&cellHeight=0.2&agentHeight=1.6&agentRadius=0.2&agentMaxClimb=0.5&agentMaxSlope=30' from origin 'https://vr.chabloz.eu' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource. 

Is the public API still available? 

Thx in advance for your answer and your wonderful work on aframe!

**Comment by donmccurdy** [comment]
Thanks! Looks like my hosting provider updated nodejs and my app wasn't compatible. Should be fixed now. 👍

**Comment by PreetiSharma15** [comment]
Why inspector recast API is throwing this error? Please help me fix this CORS issue.

![image](https://user-images.githubusercontent.com/70448973/149722027-c2686142-7dd6-4020-b74e-17d23092993e.png)

**Comment by donmccurdy** [comment]
This error may occur if the input scene or the output navmesh is too large for the size limits of my hosting provider. You can try self-hosting the tool, using a smaller/simpler scene, or using larger navmesh cells. Also see https://github.com/donmccurdy/aframe-inspector-plugin-recast/issues/25. If it doesn't seem to be any of those, you could try opening a new issue with enough details to reproduce the error.
